packages/url.lua

```diff
@@ -5,23 +5,116 @@ if pdf then SILE.require("packages/pdf") end
 
 local inputfilter = SILE.require("packages/inputfilter").exports
 
-local urlFilter = function (node, content, breakpat)
+--[[
+  FIXME: For reference, but likely too long for the SILE Manual.
```

Comment:
See other comment, but "fixme" stuff that either needs developer work or perhaps discussion should be an issue.

Lets split the rest of this up into a) code comments for technical explanation or b) documentation in the manual for anything users would benefit from knowing. I'm not at all afraid of the manual being too long. People can skip the documentation for packages that aren't interesting to them.

Reply:
Agreed, fixed.